internal/services/geobaseservice.go

```diff
@@ -0,0 +1,38 @@
+package services
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"telebot/internal/models"
+	"telebot/internal/utils"
+)
+
+// Получение ссылки из geobase
+func GetGeoUrl(wasteTypeId int, lat float64, lon float64) (string, error) {
```

Comment:
Валидный годок должен начинаться с названия типа, для которого пишется:
```suggestion
// GetGeoUrl Получение ссылки из geobase
func GetGeoUrl(wasteTypeId int, lat float64, lon float64) (string, error) {
```

Reply:
исправил